shop/views/feedback.py

```diff
@@ -26,9 +29,19 @@ def post(cls, request):
         return Response(status=status.HTTP_201_CREATED)
 
 
+def check_permissions(get_object_func):
+    def check_permissions_decorator(http_method):
+        @wraps(http_method)
+        def wrapper(self, *method_args, **method_kwargs):
+            request, pk = method_args[0], method_kwargs['pk']
+            self.check_object_permissions(request, get_object_func(pk))
+            return http_method(self, *method_args, **method_kwargs)
+        return wrapper
+    return check_permissions_decorator
```

Comment:
I think it is better to use decorator to show that the method checks permissions instead of calling `self.check_object_permissions(request, FeedbackController.get_feedback(pk))` at the beginning of each method that needs to check permissions. This decorator can be used in other views as well.
If you agree to use this decorator, also tell me what's better: use `*method_args, **method_kwargs` in `wrapper()` or just `request, pk` arguments and where to place this decorator.